recipe/bld.bat

```diff
@@ -0,0 +1,26 @@
+mkdir build
+if errorlevel 1 exit 1
+
+cd build
+if errorlevel 1 exit 1
+
+cmake .. ^
+        -G "NMake Makefiles" ^
+        -DCMAKE_BUILD_TYPE="Release" ^
+        -DCMAKE_PREFIX_PATH="%LIBRARY_PREFIX%" ^
+        -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
+        -DBUILD_SHARED_LIBS=1 ^
+        -DCMAKE_CXX_FLAGS="%CXXFLAGS% -DH5_BUILT_AS_DYNAMIC_LIB /EHsc" ^
+        -DZLIB_INCLUDE_DIR="%LIBRARY_INC%" ^
+        -DZLIB_LIBRARY="%LIBRARY_LIB%\zlib.lib" ^
```

Comment:
AFAICT it was linking to the correct `zlib` [before]( https://ci.appveyor.com/project/jakirkham/vigra-feedstock/build/1.0.13/job/kd0o2sia60x22ktp#L483 ) adding this option. So this doesn't seem to be necessary.